cmd/apiserver-boot/boot/create_resource.go

```diff
@@ -183,6 +186,8 @@ import (
 )
 
 // +genclient=true
+{{- if .NonNamespacedKind }}
+// +nonNamespaced=true{{ end }}
```

Comment:
SGTM.  We might want to move the strategy to another file (or even package) at some point, but would prefer to remain consistent until then.